docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,1344 @@
+{
```

Comment:
<p>Wow, this derivation was a lot of work! I'm surprised/impressed that the original authors went through all of this trouble!</p><p> </p><p>Thank you for working through the math</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
I have the same thought... that the authors were not afraid of the equations, and continued solving this problem.